tssc/__main__.py

```diff
@@ -100,6 +99,12 @@ def main(argv=None):
         required=True,
         help='TSSC workflow step to run'
     )
+    parser.add_argument(
```

Comment:
can we follow java standard and add a `-environment` like `java -version`? :trollface: